accounts/abi/abi_extra_test.go

```diff
@@ -31,18 +31,69 @@ func TestUnpackInputIntoInterface(t *testing.T) {
 		Amount: big.NewInt(100),
 		Memo:   []byte("hello"),
 	}
-	data, err := abi.Pack("receive", input.Sender, input.Amount, input.Memo)
+
+	rawData, err := abi.Pack("receive", input.Sender, input.Amount, input.Memo)
 	require.NoError(t, err)
 
-	// Unpack into interface
-	var v inputType
-	err = abi.UnpackInputIntoInterface(&v, "receive", data[4:]) // skips 4 byte selector
+	abi, err = JSON(strings.NewReader(TEST_ABI))
 	require.NoError(t, err)
 
-	// Verify unpacked values match input
-	require.Equal(t, v.Amount, input.Amount)
-	require.EqualValues(t, v.Amount, input.Amount)
-	require.True(t, bytes.Equal(v.Memo, input.Memo))
+	for _, test := range []struct {
+		Name                   string
```

Comment:
style nit: I think we should use unexported names when possible.

Reply:
done